assets/js/blocks/order-confirmation/downloads/style.scss

```
@@ -0,0 +1,27 @@
+.wc-block-order-confirmation-downloads {
+	.woocommerce-order-downloads table,
+	.woocommerce-order-downloads table.shop_table {
+		width: 100%;
+		border: 1px solid currentColor;
+		border-bottom: 0;
+
+		thead {
+			text-transform: uppercase;
+		}
+
+		th {
+			font-weight: bold;
+		}
+
+		th,
+		td {
+			border-style: solid;
+			border-color: currentColor;
+			border-width: 0 0 1px 0;
```

We could use the shorthand syntax here:

```suggestion
			border: 0 0 1px 0 solid currentColor;
```